webapp/sdf/src/components/list-component/list-component.stories.js

```diff
@@ -0,0 +1,14 @@
+import ListComponent from "./list-component.vue";
+
+export default {
+    title: 'ListComponent',
+    component: ListComponent,
+  };
+
+  const Template = () => ({
+    components: { ListComponent },
+    template: '<ListComponent />',
+    
+});
+  
+  export const Base = Template.bind({});
```

Comment:
Just clean code improvements.
```suggestion
  export const Base = Template.bind({});